### ISSUE TITLE
[error] - fin init (drupal8) 

### ISSUE THREAD

**Issue by ruifigueiredo** [docksal/drupal8#12]
Trying to install drupal8 on mac.

```
$ fin init

 Step 1  Initializing stack...
Removing containers...
Removing drupal8_web_1 ... done
Removing drupal8_cli_1 ... done
Removing drupal8_db_1  ... done
Removing network drupal8_default
Removing volume drupal8_project_root
Volume docksal_ssh_agent is external, skipping
Starting services...
Creating network "drupal8_default" with the default driver
Creating volume "drupal8_project_root" with local driver
Creating drupal8_db_1 ...
Creating drupal8_db_1
Creating drupal8_cli_1 ...
Creating drupal8_cli_1 ... done
Creating drupal8_web_1 ...
Creating drupal8_web_1 ... done
Waiting for drupal8_cli_1 to become ready...
Connected vhost-proxy to "drupal8_default" network.
Waiting 10s for MySQL to initialize...
 Step 2  Initializing site...
Making site directory writable...
/var/www/docroot/sites/default/settings.local.php already in place.
You are about to DROP all tables in your 'default' database. Do you want to continue? (y/n): y
Starting Drupal installation. This takes a while. Consider using the --notify global option.                                                                                                   [ok]
Drupal\Core\Config\UnmetDependenciesException: Configuration objects provided by <em class="placeholder">standard</em> have unmet dependencies: <em                                            [error]
class="placeholder">core.entity_form_display.user.user.default (field.field.user.user.user_picture)</em> in /var/www/docroot/core/lib/Drupal/Core/Config/UnmetDependenciesException.php:98
Stack trace:
#0 /var/www/docroot/core/lib/Drupal/Core/Config/ConfigInstaller.php(469): Drupal\Core\Config\UnmetDependenciesException::create('standard', Array)
#1 /var/www/docroot/core/lib/Drupal/Core/ProxyClass/Config/ConfigInstaller.php(132): Drupal\Core\Config\ConfigInstaller->checkConfigurationToInstall('module', 'standard')
#2 /var/www/docroot/core/lib/Drupal/Core/Extension/ModuleInstaller.php(141): Drupal\Core\ProxyClass\Config\ConfigInstaller->checkConfigurationToInstall('module', 'standard')
#3 /var/www/docroot/core/lib/Drupal/Core/ProxyClass/Extension/ModuleInstaller.php(83): Drupal\Core\Extension\ModuleInstaller->install(Array, false)
#4 /var/www/docroot/core/includes/install.core.inc(1572): Drupal\Core\ProxyClass\Extension\ModuleInstaller->install(Array, false)
#5 /var/www/docroot/core/includes/install.core.inc(662): install_install_profile(Array)
#6 /var/www/docroot/core/includes/install.core.inc(540): install_run_task(Array, Array)
#7 /var/www/docroot/core/includes/install.core.inc(117): install_run_tasks(Array)
#8 phar:///usr/local/bin/drush/includes/drush.inc(726): install_drupal(Object(Composer\Autoload\ClassLoader), Array)
#9 phar:///usr/local/bin/drush/includes/drush.inc(711): drush_call_user_func_array('install_drupal', Array)
#10 phar:///usr/local/bin/drush/commands/core/drupal/site_install.inc(82): drush_op('install_drupal', Object(Composer\Autoload\ClassLoader), Array)
#11 phar:///usr/local/bin/drush/commands/core/site_install.drush.inc(255): drush_core_site_install_version('standard', Array)
#12 phar:///usr/local/bin/drush/includes/command.inc(422): drush_core_site_install()
#13 phar:///usr/local/bin/drush/includes/command.inc(231): _drush_invoke_hooks(Array, Array)
#14 phar:///usr/local/bin/drush/includes/command.inc(199): drush_command()
#15 phar:///usr/local/bin/drush/lib/Drush/Boot/BaseBoot.php(67): drush_dispatch(Array)
#16 phar:///usr/local/bin/drush/includes/preflight.inc(66): Drush\Boot\BaseBoot->bootstrap_and_dispatch()
#17 phar:///usr/local/bin/drush/includes/startup.inc(458): drush_main()
#18 phar:///usr/local/bin/drush/includes/startup.inc(365): drush_run_main(false, '/', 'Phar detected. ...')
#19 phar:///usr/local/bin/drush/drush(114): drush_startup(Array)
#20 /usr/local/bin/drush(10): require('phar:///usr/loc...')
#21 {main}
```

**Comment by lmakarov** [comment]
I'm not able to reproduce this.

```
$ fin project create
Please enter the name of your project (all one word, no spaces or slashes): drupal8-test

What would you like to install?
[1] Drupal 7
[2] Drupal 8
[3] Wordpress
[4] Magento
[5] Static HTML site

: 2

Your site will be created at /Users/leonid/Work/Labs/docksal/drupal8/drupal8-test
Your site will run Drupal 8
The URL of your site will be http://drupal8-test.docksal

Do you wish to proceed? [y/n]: y
 Cloning repository...
Cloning into 'drupal8-test'...
remote: Counting objects: 42924, done.
remote: Compressing objects: 100% (4777/4777), done.
remote: Total 42924 (delta 804), reused 3901 (delta 788), pack-reused 36993
Receiving objects: 100% (42924/42924), 30.40 MiB | 3.03 MiB/s, done.
Resolving deltas: 100% (14920/14920), done.
Checking out files: 100% (13604/13604), done.
 Step 1  Initializing stack...
Removing containers...
Removing network drupal8test_default
WARNING: Network drupal8test_default not found.
Removing volume drupal8test_project_root
WARNING: Volume drupal8test_project_root not found.
Volume docksal_ssh_agent is external, skipping
Starting services...
Creating network "drupal8test_default" with the default driver
Creating volume "drupal8test_project_root" with local driver
Creating drupal8test_db_1 ... 
Creating drupal8test_db_1
Creating drupal8test_cli_1 ... 
Creating drupal8test_cli_1 ... done
Creating drupal8test_web_1 ... 
Creating drupal8test_web_1 ... done
Waiting for drupal8test_cli_1 to become ready...
Connected vhost-proxy to "drupal8test_default" network.
Waiting 10s for MySQL to initialize...
 Step 2  Initializing site...
Making site directory writable...
Copying /var/www/docroot/sites/default/settings.local.php...
You are about to DROP all tables in your 'default' database. Do you want to continue? (y/n): y
Starting Drupal installation. This takes a while. Consider using the --notify global option.                                             [ok]
Installation complete.  User name: admin  User password: 928UYTrhnZ                                                                      [ok]
Congratulations, you installed Drupal!                                                                                                   [status]

real	0m23.320s
user	0m7.680s
sys	0m3.340s
 DONE!  Open http://drupal8test.docksal in your browser to verify the setup.
```

Can you try setting up a fresh project using `fin project create`, give it a name and select option 2 for Drupal 8?

P.S. I have just pushed an update to the repo with Drupal 8.4.0 and some other minor changes. These should not really change the way the init command works, just FYI.

**Comment by ruifigueiredo** [comment]
I can create a project with D7, but with D8 I have the same error msg's.

**Comment by lmakarov** [comment]
Try running `fin update --project-images` inside the D8 project, then `fin init` again.

If that does not help, post the output from `fin sysinfo`.

**Comment by ruifigueiredo** [comment]
my sysinfo

https://pastebin.com/3xxPDeWK

**Comment by lmakarov** [comment]
You are on High Sierra... That's the problem.

TL;DR: macOS High Sierra introduced a serious bug in NFS in conjunction with the new APFS file system.

I created a tracking issue in docksal/docksal - https://github.com/docksal/docksal/issues/377
See the details there and possible workarounds and feel free to comment there. Closing this one.

**Comment by achekulaev** [comment]
I guess using Docker for Mac instead of Virtual Box is a workaround. 
https://docs.docksal.io/en/master/getting-started/env-setup-native/

**Comment by ruifigueiredo** [comment]
Thanks @achekulaev,

my steps workaround:
added DOCKER_NATIVE=1 to the file ./docksal/docksal.env

$ fin project create

fin project create
Please enter the name of your project (all one word, no spaces or slashes): testing

What would you like to install?
[1] Drupal 7
[2] Drupal 8
[3] Wordpress
[4] Magento
[5] Static HTML site

: 2

Your site will be created at /Users/rui/labdev/docksal/testing
Your site will run Drupal 8
The URL of your site will be http://testing.docksal
Do you wish to proceed? [y/n]: y
Cloning repository...
Cloning into 'testing'...
remote: Counting objects: 42924, done.
remote: Compressing objects: 100% (4778/4778), done.
remote: Total 42924 (delta 804), reused 3899 (delta 787), pack-reused 36993
Receiving objects: 100% (42924/42924), 30.40 MiB | 1.03 MiB/s, done.
Resolving deltas: 100% (14920/14920), done.
Checking out files: 100% (13604/13604), done.
Step 1  Initializing stack...
Starting services...
Creating network "testing_default" with the default driver
Creating volume "testing_project_root" with local driver
Creating testing_db_1 ...
Creating testing_cli_1 ...
Creating testing_db_1
Creating testing_cli_1 ... done
Creating testing_web_1 ...
Creating testing_web_1 ... done
Waiting for testing_cli_1 to become ready...
Waiting for testing_cli_1 to become ready...
Waiting for testing_cli_1 to become ready...
Waiting for testing_cli_1 to become ready...
Waiting for testing_cli_1 to become ready...
Waiting for testing_cli_1 to become ready...
Waiting for testing_cli_1 to become ready...
 ERROR:  testing_cli_1 heathcheck failed
         Container did not enter a healthy state within the expected amount of time.
         Try fin restart
$ cd testing
$ fin init

the setup process has been finalized.
but i can't browse the url.

$ cd ~/.docksal
$ fin reset network

then they pull the:

docksal/ssh-agent:1.0
docksal/dns:1.0
docksal/vhost-proxy:1.1

$ cd testing

$ fin restart

and I can browse the url project.
